blueprints/automation/mawaqit/Blueprint-Mawaqit_Time_Notification.yaml

```diff
@@ -0,0 +1,271 @@
+blueprint:
+  name: Mawaqit Time Notification
+  description: Notification at the time of the mawaqit prayers. PLEASE make sure to
+    install 'Mawaqit Prayer Times' integration before you use this blueprint.
+  domain: automation
+  input:
+    Mobile_to_Notify:
+      name: Mobile 1 Device to notify
+      description: Device needs to run the official Home Assistant app to receive
+        notifications.
+      default: ''
+      selector:
+        device:
+          integration: mobile_app
+    Mobile2_to_Notify:
+      name: Mobile 2 Device to notify
+      description: Device needs to run the official Home Assistant app to receive
+        notifications.
+      default: ''
+      selector:
+        device:
+          integration: mobile_app          
+          
+    Media_Player_to_Notify:
+      name: Media Player to play Athan
+      description: Choose Media player device to play the Athan.
+      selector:
+        entity:
+          domain: media_player
+    Athan_Media:
+      name: Athan Media
+      description: Name of the playlist that contains the Athan sound or mp3 link if using service music
+    Athan_Media_Fajr:
+      name: Athan Media Fajr
+      description: Name of the playlist that contains the Fajr Athan sound or mp3 link if using service music      
+    Music_Service:
+      name: Music Service
+      description: Name of the music service you are subscribed to.
+      selector:
+        select:
+          options:
+          - music
+          - APPLE_MUSIC
+          - AMAZON_MUSIC
+          - SPOTIFY
+  source_url: https://gist.github.com/shadynafie/1bc4e28717fe0955a828e8d092f7e4bf
+trigger:
+- platform: time
+  at: sensor.fajr_mawaqit
```

Comment:
can you suffix these sonsors with adhan instead mawaqit like this  sensor.fajr_adhan sensor.dhuhr_adhan ...
same for iqama sensors

Reply:
done. this needs users with old version to review their automations since we rename here the sensors